### PR TITLE
Mar names

### PR DESCRIPTION
**Pull request by rail**
should just work!

### CODE REVIEW

File: docker/funsize-update-generator/scripts/funsize.py [hunk 1]
```diff
@@ -224,9 +229,7 @@ def main():
         # if branch not set explicitly use repo-name
         mar_data["branch"] = e.get("branch",
                                    mar_data["repo"].rstrip("/").split("/")[-1])
-        mar_name = "%(appName)s-%(branch)s-%(version)s-%(platform)s-" \
-                   "%(locale)s-%(from_buildid)s-%(to_buildid)s.partial.mar" % \
-                   mar_data
+        mar_name = args.filename_template.format(**mar_data)
```
Comment (lundjordan) [inline docker/funsize-update-generator/scripts/funsize.py]: ah, right, so we can call format() on this even if it's the explicit filename (no interpolation) sent from releasetasks

File: tox.ini [hunk 2]
```diff
@@ -25,6 +25,7 @@ commands=
 max-line-length = 160
 exclude = .ropeproject,.tox,sandbox,docker,.hypothesis
 show-source = True
+ignore = E402
```
Comment (lundjordan) [inline tox.ini]: is this leftover from debugging? I can't really see where you import mid file..

File: docker/funsize-update-generator/scripts/funsize.py [hunk 3]
```diff
@@ -25,6 +25,9 @@
     "http://download.mozilla.org/",
 ]
 
+DEFAULT_FILENAME_TEMPLATE = "{appName}-{branch}-{version}-{platform}-" \
+                            "{locale}-{from_buildid}-{to_buildid}.partial.mar"
```
Comment (lundjordan) [inline docker/funsize-update-generator/scripts/funsize.py]: this default template seems to differ from what would be passed by releasetasks:
{{ product }}-{{ partial_version }}-{{ appVersion }}.{locale}.{{ buildbot2ftp(platform) }}.partial.mar

File: tox.ini (hunk 2 above)
Comment (rail) [inline tox.ini]: No, flake8 got an upgrade and doesn't like non-top imports.

File: docker/funsize-update-generator/scripts/funsize.py (hunk 3 above)
Comment (rail) [inline docker/funsize-update-generator/scripts/funsize.py]: That's OK. releasetasks set their own template, this is the default (backward compatible).

Comment (lundjordan) [inline docker/funsize-update-generator/scripts/funsize.py]: alrighty. :shipit:

**Pull request merged**